attachments.scad

```diff
@@ -1826,7 +1821,7 @@ module edge_profile(edges=EDGES_ALL, except=[], excess=0.01, convexity=10) {
 //   corner_type = Specifies how exterior corners should be formed.  Must be one of `"none"`, `"chamfer"`, `"round"`, or `"sharp"`.  Default: `"none"`
 //   size = If given the width and height of the 2D profile, will enable rounding and chamfering of internal corners when given a negative profile.
 // Side Effects:
-//   Tags the children with "remove" (and hence sets `$tag`) if no tag is already set.
```

Comment:
This side effect description is still correct, and arguably clearer to the end user.